packages/shared/src/variants/quantum.ts

```diff
@@ -49,9 +49,16 @@ class BadukHelper {
 export class QuantumGo extends AbstractGame<NewBadukConfig, QuantumGoState> {
   subgames: BadukHelper[] = [];
   quantum_stones: Coordinate[] = [];
+  
+  public sgfContent: string = '';
+  private moves: string[] = [];
+  public moveNumber: number = 0;
+
 
   constructor(config: BadukConfig) {
     super(isLegacyBadukConfig(config) ? mapToNewConfig(config) : config);
+    this.bootstrap("player Black","player white", 9, 6.5); 
```

Comment:
nit: can we name this `generateSGFHeader()` or something?  `bootstrap()` could be just about anything.